File: ImageLoader/app/src/main/java/com/lfernando/imageloader/MainActivity.java
package com.lfernando.imageloader;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.ImageView;

import com.bumptech.glide.Glide;

public class MainActivity extends AppCompatActivity {

    ImageView imageView;
    Button botao1, botao2, botao3, botao4;
    EditText input;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        botao1 = (Button) findViewById(R.id.button1);
        botao2 = (Button) findViewById(R.id.button2);
        botao3 = (Button) findViewById(R.id.button3);
        botao4 = (Button) findViewById(R.id.button4);
        final EditText input = (EditText) findViewById(R.id.editText);


        imageView = (ImageView) findViewById(R.id.imageView);

        botao1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Glide.with(getApplicationContext()).load(R.drawable.img1).into(imageView);
            }
        });

        botao2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Glide.with(getApplicationContext()).load(R.drawable.img2).into(imageView);
            }
        });

        botao3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Glide.with(getApplicationContext()).load(R.drawable.img3).into(imageView);
            }
        });

        botao4.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Glide.with(getApplicationContext()).load(input.getText().toString()).into(imageView);
            }
        });

    }

}

File: KotlinTeste/app/src/main/java/com/example/fernando/kotlinteste/MainActivity.kt
package com.example.fernando.kotlinteste

import android.app.Activity
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.annotation.IdRes
import android.view.View
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    val a: Int = 1
    val c: Int = 2
    var nome: String ?= ""
    var email: String ?= ""

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        toast("Olá mundo")
        toast("Soma $a e $c é ${a+c}")

        val campoNome = findViewById(R.id.campoNome) as EditText
        val campoEmail = findViewById(R.id.campoEmail) as EditText

        nome = campoNome.text.toString()
        email = campoEmail.text.toString()


    }

    fun Activity.toast(msg : String, duration: Int = Toast.LENGTH_SHORT){
        Toast.makeText(this, msg, duration).show()
    }

    fun click() {
        toast("Nome: $nome\nEmail: $email")

    }

}
